src/core/math/shapes/Rectangle.js

```diff
@@ -163,7 +163,7 @@ export default class Rectangle
     pad(paddingX, paddingY)
     {
         paddingX = paddingX || 0;
-        paddingY = paddingY || ((paddingY !== 0) ? paddingX : 0);
+        paddingY = paddingY || paddingX;
 
```

Comment:
Tricky one this line! This one will need to change back as if you set ```pad(10, 0)``` the paddingY would end up being set to 10 also